Makefile

```diff
@@ -50,7 +50,7 @@ build-docker:
 	docker run --rm ${USER_NS} -v "${PWD}":/go/src/github.com/hortonworks/salt-bootstrap -w /go/src/github.com/hortonworks/salt-bootstrap -e VERSION=${VERSION} golang:1.14.3 make build
 
 build-darwin:
-	GOOS=darwin go build -a -installsuffix cgo ${LDFLAGS} -o build/Darwin_x86_64/${BINARY} main.go
+	GOOS=darwin GOARCH=amd64 go build -a -installsuffix cgo ${LDFLAGS} -o build/Darwin_x86_64/${BINARY} main.go
```

Comment:
Sure, might as well add it :)

Reply:
Although since we are currently building with go@1.14.3, we won't be able to build it for Darwin-arm64, since that go version is only available for Darwin-x86_64 :/
I'll still add a make goal for it, but won't include it in the goal used to build the binaries for release.
Maybe someday we will upgrade the go version and then it can be included 👀